### DISCUSSION TITLE
"When Should I Use useFetch vs. $fetch in Nuxt 3 for SSR?"

### DISCUSSION THREAD

**Discussion by RassoMusic** [nuxt/nuxt#28848] [General]
I recently migrated from[ Vue 3 ](https://ressoapkpro.net/)to [Nuxt 3](https://ressoapkpro.net/) and have been diving into the use of useFetch and $fetch for my SSR needs. While I initially thought useFetch would cover most scenarios, I encountered some challenges, especially after upgrading to Nuxt 3.10.x.

It would be great to see more detailed guidance on when to use each approach, as understanding their distinctions is crucial for effective data fetching and performance optimization. Clear examples and best practices would really help developers like me navigate these transitions more smoothly. Keep up the great work!

**Comment by TheAlexLichter** [comment]
[This video should clear it up](https://www.youtube.com/watch?v=njsGVmcWviY)

**Comment by tag** [comment]
It _did_ help .... but it would be wonderful if this information/example was in the written documentation. I really don't care for video.
